tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -54,24 +54,14 @@ public function testFixProvider()
                 '<?php xyz($a=10, $b=20, $c=30);',
                 '<?php xyz($a=10 ,$b=20,$c=30);',
             ),
-            // test method call with multiple spaces
-            array(
+            'test method call with multiple spaces' => array(
                 '<?php xyz($a=10, $b=20, $c=30);',
                 '<?php xyz($a=10 , $b=20 ,          $c=30);',
             ),
-            // test method call with tab
-            array(
+            'test method call with tab' => array(
                 '<?php xyz($a=10, $b=20, $c=30);',
                 "<?php xyz(\$a=10 , \$b=20 ,\t \$c=30);",
             ),
-            // test method call with \n not affected
-            array(
-                "<?php xyz(\$a=10, \$b=20,\n                    \$c=30);",
-            ),
-            // test method call with \r\n not affected
-            array(
-                "<?php xyz(\$a=10, \$b=20,\r\n                    \$c=30);",
-            ),
```

Comment:
This was testing that this piece of code is not affected. Now that the fixer also fixes newlines, the piece of code *should* be affected, so I removed the test.